ts/IterableInboxMessageDisplay.tsx

```diff
@@ -83,6 +83,13 @@ const IterableInboxMessageDisplay = ({
                style={{ width: contentWidth }}
                onMessage={(event) => returnToInbox()}
                injectedJavaScript={JS}
+               onShouldStartLoadWithRequest={event => {
+                  if(event.url.slice(0,4) === 'http') {
```

Comment:
We have a similar code used in Iterable.ts
https://github.com/Iterable/react-native-sdk/blob/a469215f694b4cf00ac150150fd1e888b0fe0f9e/ts/Iterable.ts#L436-L442

It will be better to use Linking.canOpenUrl() before making the Linking library to open url externally this way.